test/e2e/example_k8petstore.go

```diff
@@ -118,30 +118,31 @@ T:
 	for {
 		select {
 		case <-transactionsCompleteTimeout:
-			Logf("Timeout %v reached, transactions not complete.  Breaking!", tick)
+			Logf("PetStore : Completion timeout %v reached, transactions not complete.  Breaking!", tick)
 			break T
 		case <-startupTimeout:
```

Comment:
Startup verifies redis works and the web app can read from it.  Transactions timeout verifies that the load generators are creating large bursts of data that aren't being lost in translation.

Hope now it Makes sense how they are orthogonal timeouts?  I should have explained this better in the comments . I'll do that :).